src/include/job.h

```diff
@@ -308,8 +308,11 @@ typedef struct resc_limit {		/* per node limits for Mom	*/
 	long long rl_vmem;		/* total mem space (virtual)	*/
 	int	  rl_naccels;		/* number of accelerators	*/
 	long long rl_accel_mem;		/* accelerator mem (real mem)	*/
+	pbs_list_head rl_oth_res;	/* list of all other resources found in execvnode and sched select*/
```

Comment:
It's good to keep variable names short, but I think we can add two characters for readability in this case. How about rl_other_res?